docs/core-definitions.md

```diff
@@ -0,0 +1,37 @@
+# Core Definitions
+
+Flow is a static analysis tool that exposes a range of global definitions to help you type your application. But flow only supports for the most part, two types of environments, node and browser and sometimes you want something a little extra that flow just doesn't support and isn't quite right to have flow support as a core system.
+
+That's where core definitions from flow-typed come into play, a home for all those global definitions to share and reuse and standardize for specific environments.
+
+To make it more concrete, lets take a look at some examples of when you might want to create to leverage a core definition.
+
+### jsx
+
+Flow is built by Meta and ships with react definitions it does not type or provide a list of available jsx properties. These can live as a core definition instead.
```

Comment:
This sentence reads a little unclearly, try tweaking?

Reply:
I think it'd be worth clarify what we are lacking that we get when adding `jsx` as an env. Most people use Flow with React and think everything is fine, so it might be unclear why it's valuable to add it.